nncf/common/pruning/model_analysis.py

```diff
@@ -219,18 +228,19 @@ def cluster_special_ops(graph: NNCFGraph, special_types: List[str], identity_typ
 class ModelAnalyzer:
     """
     Analyze the model before pruning to understand which parts could potentially be pruned without conflicts
-     (all nodes that can't get pruned input will receive a non-pruned input).
+    (all nodes that can't get pruned input will receive a non-pruned input).
 
     The algorithm consists of three steps:
-    1. Set attribute accept_pruned_input to all nodes. This attribute shows can this node potentially get
-     pruned input or node.
-    2.  Calculate can_prune attribute for all nodes by propagating accept_pruned_input up
-     (from the result of the network to the inputs). Node can be pruned if all outputs of this node accept
-      pruned input and all outputs can be pruned.
-    3. Propagates can_prune down from input nodes to the outputs.
-
-    As a result, all nodes marked by the can_prune attribute as potentially prunable or not.
+        1. Set attribute `accept_pruned_input` to all nodes. This attribute shows can this node potentially get
+        pruned input or node.
```

Comment:
... shows whether this node can potentially get pruned input

Reply:
Thanks! I fixed it.